translations/monero-core_de.ts

```diff
@@ -1316,22 +1316,22 @@ Die bisherige Wallet-Cache-Datei wird umbenannt und kann später wiederhergestel
+        <translation>Um zu verschleiern, welche Inputs in einer Monero-Transaktion ausgegeben werden, sollte ein Dritter nicht erkennen können, welche Inputs in einem Ring bereits bekannt sind. Dies würde den Schutz durch Ringsignaturen schwächen. Wenn alle bis auf einen der Inputs bereits ausgegeben sind, wird der tatsächlich ausgegebene Input sichtbar, wodurch die Wirkung von Ringsignaturen - eine der drei Hauptebenen der Privatsphäre, die Monero verwendet - zunichte gemacht wird. Um diese Inputs zu vermeiden, kann eine Liste bekannter Ausgaben verwendet werden, um die Verwendung in neuen Transaktionen zu vermeiden. Eine solche Liste wird vom Monero-Projekt verwaltet und ist auf der Website getmonero.org verfügbar, und Sie können diese Liste hier importieren.&lt;br&gt;Alternativ kannst du die Blockchain (und die Blockchain der Monero-Klone) selbst mit dem Monero-Blockchain-Blackball-Tool scannen, um eine Liste der bekannten Outputs zu erstellen.&lt;br&gt;</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="134"/>
         <source>This sets which outputs are known to be spent, and thus not to be used as privacy placeholders in ring signatures. </source>
-        <translation>Hier wird festgelegt, welche Outputs bekanntermaßen ausgegeben werden und somit nicht als Platzhalter für die Privatsphäre in Ringsignaturen verwendet werden.</translation>
+        <translation>Hier wird festgelegt, welche Outputs bekanntermaßen ausgegeben wurden und somit nicht als Verschleierungspartner für die Privatsphäre in Ringsignaturen verwendet werden.</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="135"/>
         <source>You should only have to load a file when you want to refresh the list. Manual adding/removing is possible if needed.</source>
-        <translation>Du solltest eine Datei nur dann laden müssen, wenn Sie die Liste aktualisieren möchten. Manuelles Hinzufügen/Entfernen ist bei Bedarf möglich.</translation>
+        <translation>Du solltest eine Datei nur dann laden müssen, wenn du die Liste aktualisieren möchtest. Manuelles Hinzufügen/Entfernen ist bei Bedarf möglich.</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="147"/>
         <source>Please choose a file to load blackballed outputs from</source>
-        <translation>Bitte wähle eine Datei um blackballed Outputs zu laden</translation>
+        <translation>Bitte wähle eine Datei um unzulässige Verschleierungspartner zu laden</translation>
```

Comment:
I'm not sure if this is the correct translation...

Reply:
I fiddled a long time with this and i thought it is best to describe what it is doing. A reference to the english "Blackball" can be found on the main page

1359   <translation>Oder manuell einen einzelnen Verschleierungspartner zulassen/ausschließen (Blackballen)

I'm open to suggestions though.

Reply:
Okay, then keep it in German.

Reply:
keep what in german? "Blackball"? I don't think that this is very user friendly.

The word "Blackball" just translates to "vote against" - "dagegen stimmen / ablehnen". You could translate it as "Abgelehnte Outputs" but nobody would be the wiser.

Reply:
Keep it as it is now.